bip-emilengler-redefinition-of-the-term-address.mediawiki

```diff
@@ -0,0 +1,54 @@
+<pre>
+  BIP: UNDEFINED
+  Titel: Redefinition of the term address
+  Author: Emil Engler <me@emilengler.com>
+  Status: Draft
+  Type: Informational
+  Created: 2019-10-17
+  License: Public Domain
+</pre>
+
+==Abstract==
+This BIP proposes a new term for 'address'
+
+==Specification==
+The new term is:
+''Bitcoin'' '''Invoice''' ''Address''
+Short: ''BIA''
+
+The term ''Bitcoin'' and ''Address'' are optional this has the following reasons:
+For ''Bitcoin'':
+* Useful for multicoin wallets to indicate that it belongs to Bitcoin
+* Indicates a difference between a lightning and an on-chain invoice
+For ''Address'':
+* To not confuse users with a completely new term
+* To show that it is where you send something to
+
+This gives us the four following possibilities:
+* Bitcoin Invoice Address
+* Bitcoin Invoice
+* Invoice Address
+* Invoice
+
+==Motivation==
+Bitcoin addresses are intended to be only used '''once''' and you should generate a new one for every new incoming payment.
+The term 'address' however indicates consistency because nearly everything on the internet or the offline world with the term 'address'
+is something that rarely or even never changes (postal address, e-mail address, IP-Addresses (depends heavily on the provider), etc.)
+The motivation for this BIP is to change the term address to something that indicates that the address is connected to a single transaction.
+
+==Rationale==
+The reason why we use ''Bitcoin Invoice Address'' or just ''Invoice'' is to show that it is more like a temporary thing.
+We decided to keep Bitcoin and Address as optional terms for the reasons named above.
+
+==Backwards Compatibility==
+Because this is an informational BIP it does not break any compatibility.
+However some documentation, for example BIP 21, needs to get updated.
+
+==Reference Implementation==
+''Not implemented yet''
```

Comment:
I think a list of "reference implementations" might be spammy and doesn't really help other implementations. To implement this in a given piece of software, a replacement of the term "bitcoin address" with "bitcoin invoice address" in the user facing strings should be sufficient.

Reply:
ACK